cmake/modules/LintTools.cmake

```diff
@@ -1,5 +1,4 @@
 # clang-tidy setup
-add_custom_target(run-clang-tidy-install COMMAND cp ${TTMLIR_TOOLCHAIN_DIR}/src/llvm-project/clang-tools-extra/clang-tidy/tool/run-clang-tidy.py ${TTMLIR_TOOLCHAIN_DIR}/bin/)
 add_custom_target(clang-tidy-filter-out-external-srcs COMMAND python3 ${TTMLIR_SOURCE_DIR}/tools/scripts/filter-compile-commands.py ${TTMLIR_BINARY_DIR}/compile_commands.json ${TTMLIR_SOURCE_DIR})
-add_custom_target(clang-tidy COMMAND run-clang-tidy.py -p ${PROJECT_BINARY_DIR} -warnings-as-errors '*' DEPENDS clang-tidy-filter-out-external-srcs run-clang-tidy-install)
+add_custom_target(clang-tidy COMMAND run-clang-tidy.py -p ${PROJECT_BINARY_DIR} -warnings-as-errors '*' DEPENDS clang-tidy-filter-out-external-srcs)
```

Comment:
This is dependent on your other docker change landing right?

Reply:
This is fix for for backward compatibility problem caused by new docker image.
In new image we dont have src in toolcahin folder, and filter-compile-commands.py is already installed so no need for this step.

That PR contains same change so its not depandant.

Reply:
I think I'm confused, so the new docker image is already provisioned?  I was under the impression that the existing docker still built everything inside of `${TTMLIR_TOOLCHAIN_DIR}` and that wouldn't be available until your docker file changes landed.

Reply:
No, new the image is already in use. PR is commitng Dockerfiles and updating cmake to match it

Reply:
Our image build workflow is no that advanced, it would be nice to set it up to make new image, test it, and than promote it on merge to main, but that is something I still need to work on.